rustfmt.toml

```diff
@@ -1,7 +1,9 @@
-reorder_imports  = true
-imports_granularity = "Crate"
-group_imports = "StdExternalCrate"
-wrap_comments = true
+binop_separator = "Front"
 comment_width = 100
-normalize_comments = true
+doc_comment_code_block_width = 100
```

Comment:
```suggestion
```

this is the default value

and this isn't even enabled due to

![image](https://github.com/user-attachments/assets/273005f7-ac40-4ff6-aac9-d070ec8ca42b)
https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=#binop_separator

![image](https://github.com/user-attachments/assets/df42536c-2ac2-48be-8ad1-61e75a6daa3b)
https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=#format_code_in_doc_comments